pkg/integrations/install/install_windows.go

```diff
@@ -0,0 +1,5 @@
+// +build windows
+
+package install
+
+import _ "github.com/grafana/agent/pkg/integrations/windows_exporter" // register windows_exporter
```

Comment:
It's likely that people are going to try to use the same Agent config for all their projects. Can you include this in the normal install and create a no-op integration like I did for process_exporter?